test_framework/Dockerfile.setup

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine3.19
+FROM golang:1.22.2-alpine3.19
```

Comment:
May we use [1.22-alpine3.19](https://hub.docker.com/layers/library/golang/1.22-alpine3.19/images/sha256-963da5f97ab931c0df6906e8c0ebc7db28c88d013735ae020f9558c3e6cf0580?context=explore)? So we can always get latest patch version. Thanks.